### PR TITLE
Qadir Assignment 5 Done.

### PR DESCRIPTION
**Pull request by MKadeer17**

### CODE REVIEW

File: Qadir__Work/HTML/Assignment5/QadirA5__Q1.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+.links > a::before {
+    content: "";
+    margin: 8px;
+    padding-right: 6px;
+    padding-left: 6px;
+    border: 1px solid gray;
+    background-color: rgb(0, 255, 42);
+}
```
Comment (khushboochauhan107) [inline Qadir__Work/HTML/Assignment5/QadirA5__Q1.css:66]: add a new line here

File: Qadir__Work/HTML/Assignment5/QadirA5__Q1.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                <div class="links">
+                    <a href="QadirHobbies.html" target="_blank" rel="noopener noreferrer">My Hobbies</a>
+                </div>
+                <div class="links">
+                    <a href="QadirPhotoGallery.html" target="_blank" rel="noopener noreferrer">Photo Gallery</a>
+                </div>
+            </div>
+        </div>
+    </body>
+</html>
```
Comment (khushboochauhan107) [inline Qadir__Work/HTML/Assignment5/QadirA5__Q1.html:31]: add a new line

File: Qadir__Work/HTML/Assignment5/QadirA5__Q1.css (hunk 1 above)
Comment (MKadeer17) [reply Qadir__Work/HTML/Assignment5/QadirA5__Q1.css:66]: where

File: Qadir__Work/HTML/Assignment5/QadirA5__Q1.html (hunk 2 above)
Comment (MKadeer17) [reply Qadir__Work/HTML/Assignment5/QadirA5__Q1.html:31]: why new line?

File: Qadir__Work/HTML/Assignment5/Qadir__Resume.html [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+                    <span>Programming</span>
+                    <span>Gaming</span>
+                    <span>Cricket</span>
+                    <span>Contest</span>
+                    <span>Learning New Things</span>
+                </div>
+            </div>
+        </div>
+    </body>
+</html>
```
Comment (khushboochauhan107) [inline Qadir__Work/HTML/Assignment5/Qadir__Resume.html:142]: add a new line in everypage at  last

**Review by khushboochauhan107 — APPROVED** [review]

**Review by priyaughade — APPROVED** [review]

**Pull request merged**